pkg/container/vector/vector.go

```diff
@@ -438,6 +438,59 @@ func (v *Vector) ConstExpand(m *mpool.MPool) *Vector {
 	return v
 }
 
+func (v *Vector) ConstExpandColAndNullsToFixedLength(m *mpool.MPool) *Vector {
```

Comment:
Why not just use the constExpand function?

Reply:
The ConstExpand has the following code. 
That, the ConstExpand will not extend the field `Col` of the vector when it is the constant NULL. 
But, the auto increment need to update the field `Col` of the vector when the row is NULL. 
Unfortunately, the field `Col` of the vector is NULL that is extended by the ConstExpand.

I think the ConstExpand has been used in many places. So, It is unsuitable to modify the ConstExpand itself.
And, I add the function ConstExpandColAndNullsToFixedLength.

```
if v.IsScalarNull() {
		vlen := uint64(v.ScalarLength())
		nulls.AddRange(v.Nsp, 0, vlen)
		return v
	}
```

Reply:
Functions with similar functionality should not be added. The current vector is messy enough.

Reply:
> Functions with similar functionality should not be added. The current vector is messy enough.

I have added an extra parameter in the ConstExpand and removed ConstExpandColAndNullsToFixedLength